components/ballistics.py

```diff
@@ -58,9 +59,17 @@ class BallisticsComponent:
     }
 
     FLYWHEEL_ANGLE_LOOKUP = {
-        ALGAE_MIN_DIAMETER: (math.radians(-10), math.radians(-15), math.radians(-25)),
+        ALGAE_MIN_DIAMETER: (
+            math.radians(-10) + WristComponent.COM_DIFFERENCE,
+            math.radians(-15) + WristComponent.COM_DIFFERENCE,
+            math.radians(-25) + WristComponent.COM_DIFFERENCE
+        ),
         # 16.5: (math.radians(-11.0), math.radians(-19), -25),
```

Comment:
```suggestion
```
Just in case someone gets the idea of uncommenting this and breaking the shooter